src/Engine/lib/core/StreamStorm.py

```
@@ -55,13 +55,14 @@ def init_context(self, storm_data: StormData) -> None:
         self.context.message_count = 0
         self.context.message_rate = 0.0
+        self.context.login_method = settings.login_method
         
 
     def load_cookies(self) -> None:
 
         cookies: list = get_cookies()
         
-        if not cookies and settings.login_method == "cookies":
+        if not cookies and self.context.login_method == "cookies":
             raise SystemError("Cookies not found or invalid: Try logging in again")
```

Ruff (0.15.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)